src/sentry/api/endpoints/team_alerts_triggered.py

```diff
@@ -0,0 +1,33 @@
+from django.db.models import Count
+from django.db.models.functions import TruncDay
+from rest_framework.response import Response
+
+from sentry.api.base import EnvironmentMixin
+from sentry.api.bases.team import TeamEndpoint
+from sentry.api.utils import get_date_range_from_params
+from sentry.incidents.models import IncidentTrigger, TriggerStatus
+from sentry.models import Project
+
+
+class TeamAlertsTriggeredEndpoint(TeamEndpoint, EnvironmentMixin):
+    def get(self, request, team):
+        """
+        Return a time-bucketed (by day) count of triggered alerts owned by a given team.
+        """
+        project_list = Project.objects.get_for_team_ids([team.id])
+        owner_ids = [team.actor_id] + [om.user.actor_id for om in team.member_set]
+        start, end = get_date_range_from_params(request.GET)
+        bucketed_alert_counts = list(
+            IncidentTrigger.objects.filter(
+                status=TriggerStatus.ACTIVE.value,
+                incident__alert_rule__owner__in=owner_ids,
+                incident__projects__in=project_list,
+                date_added__gte=start,
+                date_added__lte=end,
+            )
+            .annotate(bucket=TruncDay("date_added"))
+            .values("bucket")
+            .annotate(count=Count("id"))
+        )
```

Comment:
I'm not sure if this will do what you want. This will only return currently active triggers for incidents, so won't show when things triggered historically.

Could we use `IncidentActivity` here? We record all status changes there too, so we should be able to get changes from warning -> critical and so on.

Reply:
Right, yeah, switched to that model and am querying for CREATED+STATUS_CHANGE with all values except `CLOSED` so it should catch every creation, critical, and warning trigger and skip the resolutions. I forgot we resolve all but the most recent active trigger